.github/workflows/icu_ci.yml

```diff
@@ -212,6 +212,19 @@ jobs:
           make clean;
           make -j2 check
 
+  # Build and run testmap
+  testmap:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          cd icu4c/source;
+          ./runConfigureICU Linux CPPFLAGS="-DU_OVERRIDE_CXX_ALLOCATION=0";
```

Comment:
Does this need to be built with `DU_OVERRIDE_CXX_ALLOCATION` ?

Also FWIW, this doesn't quite match the docs on the page here:
http://site.icu-project.org/processes/release/tasks/integration#TOC-Build-and-run-testmap

```
$ cd <root of your ICU build tree>
$ CONFIG_FILES=test/testmap/Makefile CONFIG_HEADERS= ./config.status
$ cd test/testmap
$ make check
```

TBH, I'm not really sure though as I've never done this before.
I trust you, so the docs are probably just out-of-date. :)

Reply:
I don't know why it wold need to be built with `U_OVERRIDE_CXX_ALLOCATION` unless one is trying to cover _that_ testing here at the same time.

Reply:
The compiler flag sneaked in by copy/past, thanks for catching.